inst/include/kde1d/kde1d.hpp

```diff
@@ -612,7 +618,10 @@ Kde1d::fit_lp(const Eigen::VectorXd& x,
                                grid_points(m - 1),
                                m - 1,
                                Eigen::VectorXd::Ones(x.size()));
-    wbin = wcount.cwiseQuotient(count);
+    wbin = (count.array() == 0).select(
```

Comment:
Not following what's going on here, sorry.

Reply:
If there is a bin with no data, count and wcount for that bin is zero and `wbin = wcount.cwiseQuotient(count);` return `nan`, where it really should by 0. The updated code returns `0` in cells where `count[j] == 0` and `wcount[j] / count[j]` otherwise as required.